Formula/o/orbiton.rb

```diff
@@ -0,0 +1,28 @@
+class Orbiton < Formula
+  desc "Editor"
+  homepage "https://orbiton.zip/"
+  url "https://github.com/xyproto/orbiton.git",
+      tag:      "v2.64.1",
+      revision: "d01e9de6baa386a08dd6fca6ad76047451fbfa18"
+  license "BSD-3-Clause"
+  version_scheme 1
+  head "https://github.com/xyproto/orbiton.git"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/xyproto/orbiton").install buildpath.children
+    cd "src/github.com/xyproto/orbiton/v2" do
+      system "go", "build", "-o", "o"
+
+      bin.install "o"
+      prefix.install_metafiles
+    end
+  end
+
+  test do
+    output = shell_output("#{bin}/o --version")
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
Thanks for the informative review! I added another test that copies a file to the clipboard, writes a file from the clipboard and then compares the result.